apps/funding-service/src/db/index.spec.ts

```diff
@@ -14,6 +14,17 @@ export class MockPgInstanceSingleton {
 
   private constructor() {
     let instance = newDb();
+    instance.public.interceptQueries((sql) => {
```

Comment:
lets add a comment to what this does

Reply:
and if its used in many places, we should somehow make it re-usable so its not copy pasted